### PR TITLE
Add some additional fields to imageinspect

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Fixes: https://github.com/containers/libpod/issues/6565

File: pkg/api/handlers/types.go [hunk 1]
```diff
@@ -261,17 +262,17 @@ func ImageDataToImageInspect(ctx context.Context, l *libpodImage.Image) (*ImageI
 		//	StdinOnce:       false,
 		Env: info.Config.Env,
 		Cmd: info.Config.Cmd,
-		//	Healthcheck:     nil,
+		//Healthcheck: l.ImageData.HealthCheck,
```
Comment (mheon) [inline pkg/api/handlers/types.go:265]: Is this still commented out?

File: pkg/api/handlers/types.go [hunk 2]
```diff
@@ -261,17 +262,17 @@ func ImageDataToImageInspect(ctx context.Context, l *libpodImage.Image) (*ImageI
...
-		//	Entrypoint:      nil,
+		Volumes:    info.Config.Volumes,
+		WorkingDir: info.Config.WorkingDir,
+		Entrypoint: info.Config.Entrypoint,
 		//	NetworkDisabled: false,
 		//	MacAddress:      "",
-		//	OnBuild:         nil,
-		Labels: info.Labels,
-		//	StopSignal:      "",
+		//OnBuild:    info.Config.OnBuild,
```
Comment (mheon) [inline pkg/api/handlers/types.go:273]: Same - still commented out?

Comment (rhatdan) [reply pkg/api/handlers/types.go:273]: I am trying to figure out how to get this to work.  I would prefer to get this merged and leave the comment for someone in the future working on the TODO.

File: pkg/api/handlers/types.go (hunk 1 above)
Comment (rhatdan) [reply pkg/api/handlers/types.go:265]: I am trying to figure out how to get this to work. I would prefer to get this merged and leave the comment for someone in the future working on the TODO.

**Pull request merged**